build/Targets/Roslyn.Toolsets.Xunit.targets

```diff
@@ -18,7 +18,6 @@
     <XUnitWpfArguments>"$(OutDir)\$(AssemblyName).dll"</XUnitWpfArguments>
 
     <AutoGenerateBindingRedirects>true</AutoGenerateBindingRedirects>
-    <ForceGenerationOfBindingRedirects>true</ForceGenerationOfBindingRedirects>
```

Comment:
Is this the default now?

Reply:
Yes, `AutoGenerateBindingRedirects` just works.